hoomd/md/ForceCompositeGPU.cu

```diff
@@ -639,7 +639,10 @@ __global__ void gpu_update_composite_kernel(unsigned int N,
                                             unsigned int n_ghost,
                                             const unsigned int* d_lookup_center,
                                             Scalar4* d_postype,
+                                            Scalar4* d_velocity,
                                             Scalar4* d_orientation,
+                                            Scalar4* d_angmom,
+                                            Scalar3* d_inertia,
```

Comment:
If these are read-only, they should be const. It allows the compiler to optimize load patterns.
```suggestion
                                            const Scalar4* d_angmom,
                                            const Scalar3* d_inertia,
```